src/main/java/com/study/platform/auth/WebSecurityConfig.java

```diff
@@ -24,6 +30,8 @@ protected void configure(HttpSecurity http) throws Exception {
             .logout()
                 .logoutSuccessUrl("/").permitAll()
```

Comment:
로그아웃시 session도 지워줘야 할텐데요.. :) 
```suggestion
                .logoutSuccessUrl("/").permitAll()
                .invalidateHttpSession(true)
                .deleteCookies("JSESSIONID")
```

https://www.baeldung.com/spring-security-logout

Reply:
세윤님이 링크주신데서 확인해보니
.invalidateHttpSession(true) 는 기본값이 true라고 하네요

.deleteCookiess만 해도 될거같습니다.